internal/database/connection.go

```diff
@@ -67,14 +66,6 @@ func dbConnectionString(config *Config) string {
 	return strings.Join(p, " ")
 }
 
-// dbURI builds a Postgres URI suitable for the lib/pq driver, which is used by
```

Comment:
This isn't used anywhere in the codebase, and I lost a few minutes to it before realizing it's not used. Happy to revert this change, but I don't like dead code 😄